modules/elasticache-redis/variables.tf

```
@@ -58,3 +68,21 @@ variable "redis_clusters" {
   type        = map(any)
   description = "Redis cluster configuration"
 }
+
+variable "allow_ingress_from_vpc_stages" {
+  type        = list(string)
+  default     = ["auto", "corp"]
```

```suggestion
  default     = []
```

let's not hardcode accounts in this repo